src/fontra/client/web-components/reference-font.js

```diff
@@ -183,4 +194,142 @@ function readFileAsync(file) {
   });
 }
 
+async function getOPFSInfo() {
```

Comment:
Instead of computing whether there may be enough space to store the font, I wonder if it isn't simpler to just try to store it, and catch the error when storing fails.